### DISCUSSION TITLE
Better docs needed

### DISCUSSION THREAD

**Discussion by derwaldgeist** [material-components/material-web#5407] [General]
I started working with MWC because a client wants to include it. Coming from React and Vue, I have to admit that I am pretty stuck. The docs are pretty meagre, I couldn't even find a description what events a simple text field would fire. Or maybe I was just to dumb.

Also, once I finally made the web components show up in Vue, all were rendered with a  Times font instead of Roboto. I still haven't found out to change this. Some docs available online say that you should add Roboto's CSS to the index.html of the host app. But in our case there is no such thing (or at least, it's not easily patchable). Everything is generated via pure JavaScript.

I tried to add the font as an @import to the host component's styes. But this did not work either.

I have tried out web components for quite a number of times in the past, and always got the impression that they are interesting as a concept, but super complicated to handle and not really very practical. This library is no exception, unfortunately.

**Comment by asyncliz** [comment]
Let's start a discussion for spots where we can improve our docs. Unfortunately we don't have a tech writer, and docs usually fall to the bottom of the priority list

**Comment by treeder** [comment]
@derwaldgeist I wrote up a quick getting started guide here: https://thingster.app/things/yXPvBYZJmI4_gkygQ-3L6. I've gone through your pain many times now and mostly figured it out, so hopefully this is helpful. 

Should help you get from zero to up and running in no time.

**Reply by fjohnson** [reply]
@derwaldgeist Also, make sure you are looking at https://material-web.dev/ and not https://m3.material.io/develop/web. When I was starting out I made the mistake of paying too much attention to the latter link, which is really more for designers. Not sure why there is no link from material.io to material-web.dev... at least none that I could find. Even trying to search for the dev website is difficult!
